benchmark/benchmark.py

```
@@ -195,6 +196,35 @@ def managed(target_pid, with_bwm=False, with_perf_stat=False):
         finally:
             resource._close()
 
+def _auto_add_benchmark_sweeper() -> None:
+    """
+    Automatically add +hydra/benchmark_sweeper=<type> based on benchmark_type parameter.
+    This allows users to just specify benchmark_type instead of the full command.
+    """
+    benchmark_type = None
+    for arg in sys.argv[1:]:
+        if arg.startswith("benchmark_type="):
+            benchmark_type = arg.split("=", 1)[1]
+            break
```

Do we stay backward compatible and default to `benchmark_type=fio` here as well so we don't miss sweeping the extra params, given we already silently default to fio benchmark with `benchmark_type: "fio"`?

---

Noted, i had set it up that if you dont specify anything you have the fio benchmarks run with the default params , which would be direct io=0,but this is better.